samples/nrf9160/lwm2m_client/prj.conf

```diff
@@ -124,14 +124,6 @@ CONFIG_DATE_TIME_UPDATE_INTERVAL_SECONDS=86400
 
 CONFIG_LWM2M_VERSION_1_1=y
 CONFIG_LWM2M_PORTFOLIO_OBJ_SUPPORT=y
-CONFIG_BASE64=y
```

Comment:
Why disable SenML Json?

Reply:
I assumed you don't need this branch yourself anymore but I can put it back if you need it.